### PR TITLE
feat/cli-compare

### PR DESCRIPTION
**Pull request by FeliceGeracitano**
- Some Docs
- Compare.ts command
- Cleaner compare flow

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,9 +4,21 @@
...
+
+```bash
+$ npx bundle-checker compare \
+        --githubRepo='https://github.com/ramda/ramda.git' \
+        --installScript='yarn' \
+        --buildScript='yarn build:es' \
+        --currentBranch='CrossEye-patch-1' \
+        --distPath='dist' \
+        --targetBranch='master'
+```
```
Comment (rbelling) [inline README.md]: 👌

File: src/lib/index.ts [hunk 2]
```diff
@@ -19,62 +19,71 @@ export default class BundleChecker {
...
+        Current: ${JSON.stringify(currentSize)},
+        Target:${JSON.stringify(targetSize)}`);
     } catch (e) {
       console.log(e);
       console.log(JSON.stringify(e));
       this.spinner.fail(e);
-      return { reportText: '0' };
+      result = { reportText: '0' };
     }
+    await this.destory();
```
Comment (rbelling) [inline src/lib/index.ts]: typo

File: src/lib/index.ts [hunk 3]
```diff
@@ -19,62 +19,71 @@ export default class BundleChecker {
...
   private async init() {
-    // TODO: check if we have all the permissions
-    await this.makeFolder(this.workDir);
+    if (this.inputParams.githubRepo) {
+      this.workDir = this.generateWorkDirName();
+      await exec(`mkdir -p ${this.workDir}`);
+      process.chdir(this.workDir);
+      const { stdout } = await exec(`pwd`);
+      console.log(`PWD: ${stdout}`);
+      await this.cloneRepo(this.inputParams.githubRepo);
```
Comment (rbelling) [inline src/lib/index.ts]: 👌

File: src/lib/index.ts [hunk 4]
```diff
@@ -19,62 +19,71 @@ export default class BundleChecker {
...
+    }
   }
 
-  // private printResult(current: string, target: string) {
-  //   this.spinner.info(`Current: ${printBytes(current)}`);
-  //   this.spinner.info(`Target: ${printBytes(target)}`);
-  //   this.spinner.info(`Diff: ${printBytes(parseInt(current, 10) - parseInt(target, 10))}`);
-  // }
+  private async destory() {
+    if (this.inputParams.githubRepo) {
```
Comment (rbelling) [inline src/lib/index.ts]: minor: could we rename this to `gitRepository` ?

Comment (FeliceGeracitano) [reply src/lib/index.ts]: yes

**Comment by FeliceGeracitano** [comment]
- [] destroy Typo
- [] `gitRepository `

**Pull request merged**